.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 python:
   - "2.7"
 jdk:
-  - oraclejdk8
+  - oraclejdk7
```

Comment:
I think you can rely on graal-core and standard development environment for jdk8 testing.